setup.cfg

```diff
@@ -43,6 +43,7 @@ install_requires =
     cmarkgfm==0.6.* #Update regularly
```

Comment:
This is pretty old now. Might be worthwhile looking as to whether a newer version is approriate.